modules/core/keeper/msg_server.go

```diff
@@ -33,24 +33,10 @@ func (k Keeper) CreateClient(goCtx context.Context, msg *clienttypes.MsgCreateCl
 		return nil, err
 	}
 
-	clientID, err := k.ClientKeeper.CreateClient(ctx, clientState, consensusState)
-	if err != nil {
+	if _, err = k.ClientKeeper.CreateClient(ctx, clientState, consensusState); err != nil {
```

Comment:
Since the returned `cliendID` is not used anymore, should we remove it from the return values of `CreateClient`? Although if we keep it, we could also add it to the `MsgCreateClientResponse`, which is an empty `struct` at the moment...

Reply:
I was debating the same question my self. The clientID is used in the testing cases so I assumed it is necessary. Obviously, it is also an identifier, which we want track and compare and include in the emission, so I would keep it.
Regardless, I would leave to the experts :-), whatever you guys think!

Reply:
we should leave as is. I like the idea of adding it to `MsgCreateClientResponse`